Classes/Views/IssueTextActionsView+Markdown.swift

```diff
@@ -31,7 +31,24 @@ extension IssueTextActionsView {
             IssueTextActionOperation(icon: UIImage(named: "bar-ul"), operation: .line("- ")),
             IssueTextActionOperation(icon: UIImage(named: "bar-indent"), operation: .line("  ")),
             IssueTextActionOperation(icon: UIImage(named: "bar-link"), operation: .wrap("[", "](\(UITextView.cursorToken))")),
-            ]
+            IssueTextActionOperation(icon: UIImage(named: "cloud-upload"), operation: .execute({ [weak viewController] in
+                guard let strongVc = viewController else { return }
+                
+                let imagePicker = UIImagePickerController()
+                imagePicker.delegate = strongVc as? (UINavigationControllerDelegate & UIImagePickerControllerDelegate)
```

Comment:
Neat!

Reply:
@Sherlouk should we update the `viewController` param type so this is explicit? Otherwise this will silently break, right?

Reply:
Yeah my mistake, was building this for bug report only originally but moved it after a merge conflict 😅 Will fix